src/vs/workbench/contrib/search/browser/searchModel.ts

```diff
@@ -247,6 +258,10 @@ export class CellMatch {
 	}
 
 
+	setCellModel(cell: ICellViewModel) {
```

Comment:
The idea is that the model for the cell will be `CellSearchModel` until it's opened. Then, you can set the proper `ICellViewModel`. `CellSearchModel` just has simple info from deserialization like inputs.